docs/source/index.rst

```diff
@@ -22,14 +22,6 @@ Apache Arrow DataFusion
 Table of Contents
 =================
 
-.. _toc.usage:
-
-.. toctree::
-   :maxdepth: 1
-   :caption: Supported Environments
-
-   Rust <https://docs.rs/crate/datafusion/>
```

Comment:
This is the very first link on the page and takes the user to a completely different web site. I moved it down into the user guide.